ui/raidboss/data/05-shb/eureka/zadnor.js

```diff
@@ -81,30 +133,1155 @@ export default {
+      netRegex: NetRegexes.startsUsing({ source: 'Stormborne Zirnitra', id: '5E54' }),
+      condition: (data) => data.ce === 'serpents',
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 5,
+      alertText: (data, _matches, output) => {
+        data.serpentsTurbineCount = (data.serpentsTurbineCount || 0) + 1;
+        // TODO: how does this loop?
+        if (data.serpentsTurbineCount === 1)
+          return output.knockbackDonut();
+        else if (data.serpentsTurbineCount === 2)
+          return output.knockbackIntoCircle();
+        else if (data.serpentsTurbineCount === 3)
+          return output.knockbackExplosion();
+        else if (data.serpentsTurbineCount === 4)
+          return output.knockbackDonut();
+        else if (data.serpentsTurbineCount === 5)
+          return output.knockbackIntoSafe();
+      },
+      outputStrings: {
+        knockbackDonut: {
+          en: 'Knockback + Stack Donuts Middle',
+        },
+        knockbackIntoCircle: {
+          en: 'Knockback (towards first circles)',
+        },
+        knockbackIntoSafe: {
+          en: 'Knockback (towards open spots)',
+        },
+        knockbackExplosion: {
+          // Can't trust people to make a safe spot,
+          // so using knockback prevention is probably the best advice.
+          en: 'Knockback (prevent)',
+        },
+      },
+    },
     // ***** Feeling the Burn *****
+    {
+      id: 'Zadnor Feeling Suppressive Magitek Rays',
+      netRegex: NetRegexes.startsUsing({ source: 'Blackburn', id: '5C40', capture: false }),
+      condition: (data) => data.ce === 'feeling',
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Zadnor Feeling Chain Cannon You',
+      netRegex: NetRegexes.headMarker({ id: '00A4' }),
+      condition: (data, matches) => data.ce === 'feeling' && data.me === matches.target,
+      alertText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Chain Cannon on YOU',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Feeling Chain Cannon Not You',
+      netRegex: NetRegexes.headMarker({ id: '00A4', capture: false }),
+      condition: (data) => data.ce === 'feeling',
+      delaySeconds: 3,
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Stay Out of Lasers',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Feeling Analysis',
+      netRegex: NetRegexes.startsUsing({ source: 'Blackburn', id: '5C37', capture: false }),
+      condition: (data) => data.ce === 'feeling',
+      run: (data) => data.feelingAnalysis = true,
+    },
+    {
+      id: 'Zadnor Feeling Read Orders Coordinated Assault',
+      netRegex: NetRegexes.startsUsing({ source: 'Blackburn', id: '5C34', capture: false }),
+      condition: (data) => data.ce === 'feeling',
+      alertText: (data, _matches, output) => {
+        if (data.feelingAnalysis)
```

Comment:
Unrelated to this, and not commenting for an update, but should these sorts of logic statements be turned into ternary operators? How do you feel about that?

Reply:
Sounds good.  Fixed them up.  I left the responses alone because they're long in both a lint and a readability sense.